### PR TITLE
Create QuickSort.js

### PR DESCRIPTION
**Pull request by Dhaxor**
Added Quick sort

### CODE REVIEW

File: SortingAlgorithms/QuickSort.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+var items = [5,3,7,6,2,9];
```
Comment (ignacio-chiazzo) [inline SortingAlgorithms/QuickSort.js:1]: remove this

File: SortingAlgorithms/QuickSort.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        }
+        if (index < right) { //more elements on the right side of the pivot
+            quickSort(items, index, right);
+        }
+    }
+    return items;
+}
+// first call to quick sort
+var sortedArray = quickSort(items, 0, items.length - 1);
+console.log(sortedArray); //prints [2,3,5,6,7,9]
```
Comment (ignacio-chiazzo) [inline SortingAlgorithms/QuickSort.js:42]: Could you remove these examples and add them to a file within the `test` folder? It could use a few test cases.

**Review by ignacio-chiazzo — COMMENTED** [review]
Great! Thanks for adding it.

File: SortingAlgorithms/QuickSort.js (hunk 1 above)
Comment (Dhaxor) [reply SortingAlgorithms/QuickSort.js]: ok. Got it.

File: SortingAlgorithms/QuickSort.js (hunk 2 above)
Comment (Dhaxor) [reply SortingAlgorithms/QuickSort.js]: ok

**Pull request closed without merging**